sw/device/silicon_creator/rom/BUILD

```diff
@@ -159,9 +160,18 @@ opentitan_rom_binary(
     name = "rom_with_fake_keys",
     testonly = True,
     deps = [
-        ":linker_script",
-        ":rom_common",
-        ":rom_start",
+        ":rom_without_keys",
+        "//sw/device/silicon_creator/rom/keys/fake",
+        "//sw/device/silicon_creator/rom/keys/fake/spx:fake",
+    ],
+)
+
+opentitan_rom_binary(
+    name = "rom_with_real_keys",
```

Comment:
Yeah, we also talked about this with @cfrantz. I prefer the second option since we can't forget updating the hash in the test -- assuming that it's ok to require a build.

Reply:
Sounds good. We should also think about how do we want to enable integration testing on the FPGA post ROM release, as we want to freeze the ROM for testing purposes and start identifying workarounds in software for any issues (in preparation for post-silicon). Maybe the best option is to pin a release bitstream. Something to discuss as part of this. 

re. Relaying on the test, we'll have to make sure it is a test that runs in a reasonable amount of time so that we can make it part of gate level simulation. This assumes that the secure environment will be able to run Bazel and all the dependencies. @msfschaffner should be able to confirm this.

Reply:
I also prefer the second option. I will confirm with Nuvoton RE Bazel.

Reply:
Resolving this since @timothytrippel started working on it.